docs/Homebrew-and-Python.md

```diff
@@ -2,18 +2,16 @@
 
 This page describes how Python is handled in Homebrew for users. See [Python for Formula Authors](Python-for-Formula-Authors.md) for advice on writing formulae to install packages written in Python.
 
-Homebrew should work with any [CPython](https://stackoverflow.com/questions/2324208/is-there-any-difference-between-cpython-and-python) and defaults to the macOS system Python.
-
-Homebrew provides formulae to brew Python 3.y. A `python@2` formula was provided until the end of 2019, at which point it was removed due to the Python 2 deprecation.
```

Comment:
`Homebrew provides formulae to brew Python 3.y.` feels like it may still be useful here?

Reply:
Agree, maybe we can say "Homebrew provides formulae for Python 3. Older versions of Python are not supported."?

("Python 3" is the idiomatic-ish way to refer to `3.x`)